package.json

```diff
@@ -39,6 +39,7 @@
         "restify-clients": "2.6.7",
         "restify-errors": "^5.0.0",
         "sshpk": "1.16.1",
+        "storinfo": "git+https://github.com/joyent/node-storinfo.git",
```

Comment:
Whether `node-storinfo` gets put on npm is a different discussion, but I think at the very least `node-storinfo` should push a `git tag` and then this `package.json` should use that tag/version.

Reply:
Done - I tagged the commit in node-storinfo and then updated the package.json here to point to it.